### ISSUE TITLE
Platform selection in the beginner tutorial is broken

### ISSUE THREAD

**Issue by actgardner** [pachyderm/pachyderm#4881]
**Provide a link to the page with an issue:**
https://docs.pachyderm.com/latest/getting_started/beginner_tutorial/#adding-data-to-pachyderm

**Describe what is wrong:**
The tabbed platform selector is rendering as a series of radio buttons.

<img width="398" alt="Screen Shot 2020-05-13 at 11 13 23 AM" src="https://user-images.githubusercontent.com/3100188/81832209-3a640100-950c-11ea-83b6-9acbb55b7168.png">

**Describe the suggested changes:**

**Anything else we need to know?**

**Environment:**
- Kubernetes version (use `kubectl version`):
- Pachyderm CLI and pachd server version (use `pachctl version`):
- Cloud provider (e.g. aws, azure, gke) or local deployment (e.g. minikube vs dockerized k8s):
- OS:
- Other:

**Comment by JoeyZwicker** [comment]
That's weird! Thanks!

**Comment by svekars** [comment]
The syntax in the Python Markdown extension `superfences` that we used for tab markup has been deprecated in favor of another extension called `tabbed` in `pymarkdown-extensions` v7.0. I've tried to  change it today but it doesn't seem to work. Probably, I could try to downgrade to a previous version to fix this. There is also a possibility that some of the other pymarkdown extensions that we use conflicts with `tabbed`. I’ll investigate more.
